internal/templates/element/base.go

```diff
@@ -32,6 +32,28 @@ const {{.ElementName}}Limbs = {{.NbWords}}
 // {{.ElementName}}Bits number bits needed to represent {{.ElementName}}
 const {{.ElementName}}Bits = {{.NbBits}}
 
+// ToBytes converts z into a slice of byte
+// the resulting slice is seen as a big endian integer
+func (z *{{.ElementName}}) ToBytes() []byte {
+	var _z {{.ElementName}}
+	_z.Set(z).FromMont()
+	res := make([]byte, {{.ElementName}}Limbs*8)
+	binary.BigEndian.PutUint64(res[({{.ElementName}}Limbs-1)*8:], _z[0])
+	for i := {{.ElementName}}Limbs - 2; i >= 0; i-- {
+		binary.BigEndian.PutUint64(res[i*8:(i+1)*8], _z[{{.ElementName}}Limbs-1-i])
+	}
+	return res
+}
+
+// SetBytes sets z form a byte array
+// e is seen as a big endian unsigned integer
```

Comment:
Same remark for the doc: "is seen as a big..." --> something clsoer to `SetBytes interprets e as the bytes of a big-endian unsigned integer, sets z to that value, and returns z.`

Reply:
(+ rename `e` to `buf` )